app/controllers/nonsipp/bondsdisposal/BondsDisposalListController.scala

```diff
@@ -80,14 +80,22 @@ class BondsDisposalListController @Inject()(
             BadRequest(view(errors, viewModel(srn, page, bondsList, mode, request.userAnswers)))
           }.merge
         },
-        answer =>
-          Redirect(
-            navigator.nextPage(
-              BondsDisposalListPage(srn, answer),
-              mode,
-              request.userAnswers
-            )
-          )
+        answer => {
+          val inProgressUrl = request.userAnswers
```

Comment:
This continues with incomplete journey for the selected base entity. FYI @jamalosman